_data/navigation.yml

```diff
@@ -16,7 +16,9 @@
 - text: AirCasting
   url: /aircasting
   subitems:
-    - text: Aircasting App
+    - text: AirCasting Maps
+      url: http://aircasting.habitatmap.org/mobile_map
```

Comment:
Please use `/map`. That way we leave AirCasting responsible to choose which map to choose. At the moment that means the mobile map but we could change our minds in the future.